cli/azd/cmd/testdata/TestUsage-azd-template-list.snap

```diff
@@ -5,8 +5,9 @@ Usage
   azd template list [flags]
 
 Flags
-        --docs 	: Opens the documentation for azd template list in your web browser.
-    -h, --help 	: Gets help for list.
+        --docs          	: Opens the documentation for azd template list in your web browser.
+    -h, --help          	: Gets help for list.
+    -s, --source string 	: Filter templates by source
```

Comment:
```suggestion
    -s, --source string 	: Filters templates by source.
```

Reply:
Should it be `Filters` and a dot at the end?

Commenting here.. but, to be changed in the source